nasa-veda-platform/adding-content/docs-and-notebooks.qmd

```diff
@@ -67,7 +67,7 @@ Something like: public.ecr.aws/nasa-veda/pangeo-notebook:60b023fba2ca5f9e19d285c
 
 To give the notebooks a standard look and feel we typically include the following sections:
 
-- **Run this Notebook**: The section explains how to run the notebook locally, on VEDA JupyterHub or on [mybinder](https://mybinder.org/). There are several examples of what this section can look like in the template notebooks.
+- **Run this Notebook**: The section explains how to run the notebook locally, on [VEDA JupyterHub](https://hub.openveda.cloud/), on [VEDA Binder](https://binder.openveda.cloud/), or on the public [MyBinder](https://mybinder.org/). There are several examples of what this section can look like in the template notebooks.
```

Comment:
should the jupyterhub link be an internal reference to https://docs.openveda.cloud/nasa-veda-platform/scientific-computing/ ?
Or something that explains the Hub? Most people with hub access won't need the binder links.

Reply:
I'm also concerned about link to https://binder.openveda.cloud if we don't have guardrails, since it might be abused by non-veda related work. @yuvipanda